src/sentry/integrations/github/integration.py

```diff
@@ -144,17 +144,23 @@ def reinstall(self) -> None:
         self.reinstall_repositories()
 
     def message_from_error(self, exc: Exception) -> str:
-        # TODO(mgaeta): Clean up the conditional flow.
-        if isinstance(exc, ApiError):
-            message = API_ERRORS.get(exc.code, "")
-            if exc.code == 404 and re.search(r"/repos/.*/(compare|commits)", exc.url):
-                message += f" Please also confirm that the commits associated with the following URL have been pushed to GitHub: {exc.url}"
+        if not isinstance(exc, ApiError):
+            return ERR_INTERNAL
 
-            if not message:
-                message = exc.json.get("message", "unknown error") if exc.json else "unknown error"
-            return f"Error Communicating with GitHub (HTTP {exc.code}): {message}"
+        if not exc.code:
```

Comment:
I want to point out that this is a behavior change, previously falsy values were looked up in `API_ERRORS`. I assume that you needed to fix a typing error here?

Reply:
Yeah I was getting an errors saying that I could not look up an `Optional[str]` in a `Mapping[str, str]`.